### PR TITLE
Upgrade to latest version of Error Prone

### PR DESCRIPTION
**Pull request by meltsufin**
Related to #1791.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ git:
   depth: 9999999
 jdk:
   - oraclejdk8
+  - oraclejdk13
```
Comment (dmitry-s) [inline .travis.yml]: is it allowed to have multiple versions here?

Comment (meltsufin) [reply .travis.yml]: yes, and there would be a build for each

**Comment by meltsufin** [comment]
Still doesn't work on JDK 13.
```
Caused by: java.lang.NoSuchMethodException: com.sun.tools.javac.comp.Resolve.findIdent(com.sun.tools.javac.comp.Env,com.sun.tools.javac.util.Name,com.sun.tools.javac.code.Kinds$KindSelector)
```

**Comment by ttomsu** [comment]
Is this PR still open and relevant?

**Pull request merged**